app/views/admin/petitions/edit_response.html.erb

```diff
@@ -18,6 +18,8 @@
   <% end -%>
 <% end -%>
 
+<%= link_to("Publish Debate Schedule", edit_scheduled_debate_date_admin_petition_path(@petition)) %>
+
 <%= form_for [:admin, @petition ], :url => update_response_admin_petition_path(@petition), :builder => AdminFormBuilder do |f| -%>
```

Comment:
With #145 I link to the show page instead of the edit page and put the possible actions in a nav list there.  When #145 is merged and you rebase you'll want to move this link.  (Obviously you couldn't have predicted that, just a heads up for potential rebase re-work).